dbt/include/spark/macros/materializations/table.sql

```diff
@@ -0,0 +1,32 @@
+{% materialization table, adapter = 'spark' %}
+  {%- set identifier = model['alias'] -%}
+
+  {%- set non_destructive_mode = (flags.NON_DESTRUCTIVE == True) -%}
```

Comment:
we're ripping out `--non-destructive` in 0.14.0; you can remove all relevant code from this PR :)